AttackItem.h

```diff
@@ -1 +1,10 @@
-#include "Item.h"
\ No newline at end of file
+#pragma once
+
+#include "Item.h"
+#include "Character.h"
+
+struct AttackItem : Item
+{
+    AttackItem() : Item("attack item", 3) { }
```

Comment:
Re-read the instructions that say how much boost the AttackItem should provide
- [ ] revise